SDWebImage/SDWebImageManager.m

```diff
@@ -202,9 +200,7 @@ - (void)diskImageExistsForURL:(nullable NSURL *)url
                     
                     BOOL cacheOnDisk = !(options & SDWebImageCacheMemoryOnly);
 
-                    if (options & SDWebImageRefreshCached && cachedImage && !downloadedImage) {
```

Comment:
This is used previously with that `SDWebImageDownloaderIgnoreCachedResponse`. But now downloadedImage should not be nil if network is OK. So this logic is useless.